flaml/nlp/huggingface/utils.py

```diff
@@ -2,12 +2,12 @@
 from itertools import chain
 import numpy as np
 
-from ...data import (
-    SUMMARIZATION,
-    SEQREGRESSION,
+from flaml.automl.task import (
```

Comment:
The ...data is meant to be kept as the relative path to be consistent with the other files. Is there any reasons to change it to the absolute path?